head.html

```diff
@@ -1,4 +1,4 @@
 <meta name="viewport" content="width=device-width, initial-scale=1"/>
-<script src="/libs/scripts/scripts.js" type="module" crossorigin="use-credentials"></script>
+<script src="/libs/scripts/scripts.js" type="module"></script>
```

Comment:
I don't remember the details, but this file was previously heavily cached and making changes to it lead to various issues. Are you certain this should be removed? Why do Milo's scripts interfere with Merch logic?

Reply:
is the use case to render a milo.adobe.com page with different commerce source, @yesil ? may be we could make that change a separate one as i feel like it's not going to be an easy one and we can still play around with most commerce pages on CC & DC

Reply:
With `crossorigin="use-credentials"` all the subsequent js module loaded by `/libs/scripts/scripts.js` will be subject to CORS restrictions. Since `maslibs` param will allow to load commerce.js from a different domain instead of libs/deps folder, this will fail.  With `crossorigin="use-credentials"` we cannot use `Access-Control-Allow-Origin:*` on commerce.js neither, it must be an explicit domain, without wildcards.

I don't know changing this file will causes an immediate cache flush, but we don't need it immediately. We can start using it once CDN caches are naturally refreshed.

Reply:
we discussed this offline and agreed we can move this to a second one line PR

Reply:
Moved this change in a separate PR: https://github.com/adobecom/milo/pull/2549